user/e16/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "e16"
+pkgver = "1.0.30"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--prefix=/usr",
+    "--sysconfdir=/etc",
+    "--mandir=/usr/share/man",
+    "--localstatedir=/var",
+    "--enable-sound",
+    "--enable-sound-alsa",
+    "--enable-sound-pulse",
+]
+hostmakedepends = ["automake", "libtool", "pkgconf"]
+makedepends = [
+    "alsa-lib-devel",
+    "audiofile-devel",
+    "dbus-devel",
+    "fontconfig-devel",
+    "freetype-devel",
+    "gdk-pixbuf-devel",
+    "gettext-devel",
+    "glib-devel",
+    "imlib2-devel",
+    "libice-devel",
+    "libpulse-devel",
+    "libsm-devel",
+    "libsndfile-devel",
+    "libx11-devel",
+    "libxcomposite-devel",
+    "libxcursor-devel",
+    "libxdamage-devel",
+    "libxext-devel",
+    "libxfixes-devel",
+    "libxft-devel",
+    "libxinerama-devel",
+    "libxpm-devel",
+    "libxpresent-devel",
+    "libxrandr-devel",
+    "libxrender-devel",
+    "libxres-devel",
+    "libxxf86vm-devel",
+    "pango-devel",
+    "xbitmaps",
+    "xorgproto",
+]
+depends = ["desktop-file-utils", "python"]
+pkgdesc = "Themed window manager for X11"
+maintainer = "rane <rane+chimera@junkyard.systems>"
+license = "BSD-2-Clause"
+url = "https://enlightenment.org/e16"
+source = f"https://downloads.sourceforge.net/enlightenment/e16/{pkgver}/e16-{pkgver}.tar.gz"
```

Comment:
```suggestion
source = f"$(SOURCEFORGE_SITE)/enlightenment/e16/{pkgver}/e16-{pkgver}.tar.xz"
```

Reply:
OK thanks I've changed it, that's useful to know.

Reply:
note also the .xz on the end; they make one here and it's purely smaller :)